linters/nancy/plugin.yaml

```diff
@@ -40,4 +40,8 @@ lint:
           list: ["${runtime}", "${linter}", "${env.PATH}"]
         - name: USER
           value: ${env.USER}
+          optional: true
+        - name: USERNAME
```

Comment:
I think we want this flipped? It requires USER but USERNAME might be set in the env